src/useBaseAxios.ts

```diff
@@ -62,14 +63,18 @@ function useBaseAxios<Data>(
     source = axios.CancelToken.source();
   };
 
+  const refetch = () => {
+    getData();
+  };
```

Comment:
What about just doing this in the return statement?

```js
return [getData, { cancel, data, error, loading, refetch: getData }];
```

Reply:
clean

Reply:
Wouldn't it be

```js
return [getData, { cancel, data, error, loading, getData: refetch }];
```